spring-cloud-gcp-examples/spring-cloud-gcp-trace-example/README.adoc

```diff
@@ -3,23 +3,41 @@
 This sample application demonstrates using the
 link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-trace[Spring Cloud GCP Trace Starter] in your code.
 
+You can read about the Spring Boot Starter for Stackdriver Trace in more detail at the
+https://docs.spring.io/spring-cloud-gcp/docs/1.0.0.BUILD-SNAPSHOT/reference/htmlsingle/#_spring_cloud_sleuth[Spring
+Cloud GCP Reference Document Stackdriver Trace section].
+
 == Setup & Configuration
 1. Create a Google Cloud Platform Project
 1. https://cloud.google.com/docs/authentication/getting-started#creating_the_service_account[Create a service account]
-   with Trace Append permission. Furnish a new JSON key and then set the credentials using the
-   `GOOGLE_APPLICATION_CREDENTIALS` environment variable or
-   link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-core/README.adoc[using GCP Starter Core properties].
+with Trace Append permission. Furnish a new JSON key and then set the credentials using the
+`GOOGLE_APPLICATION_CREDENTIALS` environment variable or
+link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-core/README.adoc[using GCP Starter Core properties].
```

Comment:
Hm. There is some inconsistency with links.
The previous one is from "https://docs.spring.io/spring-cloud-gcp/docs/1.0.0.BUILD-SNAPSHOT/" - absolute value from the Internet. This one is relative to the current document state in the project. But both of them are related to the same SC-GCP document. 

Am I missing anything? :shrug:

OTOH in other projects we definitely use [ASCII Doc](http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#links) (section `Internal cross references`) style for linking. For example Spring Kafka: https://raw.githubusercontent.com/spring-projects/spring-kafka/master/src/reference/asciidoc/whats-new.adoc

Pay attention to the style like: `See <<message-listeners>> for more information.`.

Reply:
We're linking from a README to the reference document, can't use internal links.

Reply:
OK. I got it. Thanks.
Never mind